charts/incubator/bar-assistant/values.yaml

```
@@ -0,0 +1,47 @@
+image:
+  repository: barassistant/server:v3
```

```suggestion
  repository: barassistant/server
```
Tag gets added in the tag field